source/ProcessManager.Orchestrations/Processes/BRS_021/ForwardMeteredData/V1/BusinessValidation/PeriodValidationRule.cs

```diff
@@ -12,49 +12,70 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using Energinet.DataHub.ProcessManager.Components.Abstractions.ValueObjects;
 using Energinet.DataHub.ProcessManager.Components.BusinessValidation;
 using Energinet.DataHub.ProcessManager.Components.BusinessValidation.Validators;
+using Energinet.DataHub.ProcessManager.Orchestrations.Processes.BRS_021.ForwardMeteredData.V1.Extensions;
 using Energinet.DataHub.ProcessManager.Orchestrations.Processes.BRS_021.ForwardMeteredData.V1.Model;
 using Energinet.DataHub.ProcessManager.Orchestrations.Processes.BRS_021.Shared.ElectricityMarket.Extensions;
 using NodaTime;
 
 namespace Energinet.DataHub.ProcessManager.Orchestrations.Processes.BRS_021.ForwardMeteredData.V1.BusinessValidation;
 
-/// <summary>
-/// TODO: Implement correct validation rule
-/// "Stub" period validation rule, used to verify that validation works for <see cref="ForwardMeteredDataBusinessValidatedDto"/>.
-/// </summary>
 public class PeriodValidationRule(PeriodValidator periodValidator)
         : IBusinessValidationRule<ForwardMeteredDataBusinessValidatedDto>
 {
     public const int MaxAllowedPeriodAgeInYears = 3;
 
-    /// <summary>
-    /// TODO: Add correct error messages.
-    /// "Stub" error message used to very that validation works.
-    /// </summary>
+    public const int MinimalPeriodSizeInHours = 4;
```

Comment:
```suggestion
    private const int MinimalPeriodSizeInHours = 4;
```

MaxAllowedPeriodAgeInYears can also be private